apps/spikeConverter.cpp

```diff
@@ -29,11 +43,52 @@ int main( int argc, char* argv[] )
     }
 
     lunchbox::Clock clock;
-    const brion::SpikeReport in( brion::URI( argv[1] ), brion::MODE_READ );
+    brion::SpikeReport in( brion::URI( argv[1] ), brion::MODE_READ );
     const float readTime = clock.resetTimef();
 
     brion::SpikeReport out( brion::URI( argv[2] ), brion::MODE_WRITE );
-    out.writeSpikes( in.getSpikes( ));
+
+    if( in.getReadMode() == brion::SpikeReport::STREAM )
+    {
+        // Stream-to-File conversion
+        while( in.getNextSpikeTime() != brion::UNDEFINED_TIMESTAMP )
+        {
+            in.waitUntil( brion::UNDEFINED_TIMESTAMP, STREAM_READ_TIMEOUT_MS );
+            out.writeSpikes( in.getSpikes( ));
+        }
```

Comment:
Correct, the spikes were being written multiple times and a clear() solved the problem, thanks!